src/components/site-form.tsx

```diff
@@ -292,20 +292,29 @@ export const SiteForm = ( {
 											</div>
 										</Button>
 									</div>
-									{ isAdvancedSettingsVisible && (
-										<label className="flex flex-col gap-1.5 leading-4">
-											<span onClick={ onSelectPath } className="font-semibold">
-												{ __( 'Local path' ) }
-											</span>
-											<FormPathInputComponent
-												isDisabled={ isPathInputDisabled }
-												doesPathContainWordPress={ doesPathContainWordPress }
-												error={ error }
-												value={ sitePath }
-												onClick={ onSelectPath }
-											/>
-										</label>
-									) }
+									<div
+										className={ cx(
+											'transition-all duration-500 ease-in-out overflow-hidden',
+											isAdvancedSettingsVisible
+												? 'max-h-96 opacity-100 mb-6'
+												: 'max-h-0 opacity-0 mb-0'
+										) }
+									>
+										{ isAdvancedSettingsVisible && (
+											<label className="flex flex-col gap-1.5 leading-4 p-2">
+												<span onClick={ onSelectPath } className="font-semibold">
+													{ __( 'Local path' ) }
+												</span>
+												<FormPathInputComponent
+													isDisabled={ isPathInputDisabled }
+													doesPathContainWordPress={ doesPathContainWordPress }
+													error={ error }
+													value={ sitePath }
+													onClick={ onSelectPath }
+												/>
+											</label>
+										) }
```

Comment:
What's the benefit of removing the component?

Here is how it looks if we leave it:


https://github.com/user-attachments/assets/17210cdd-c633-47bd-8a7e-3dad5def02ee



```suggestion
										<label className="flex flex-col gap-1.5 leading-4 p-2">
											<span onClick={ onSelectPath } className="font-semibold">
												{ __( 'Local path' ) }
											</span>
											<FormPathInputComponent
												isDisabled={ isPathInputDisabled }
												doesPathContainWordPress={ doesPathContainWordPress }
												error={ error }
												value={ sitePath }
												onClick={ onSelectPath }
											/>
										</label>
```

Reply:
Sounds good, let's keep it. I made the changes in https://github.com/Automattic/studio/pull/390/commits/54a8e97014783b3738ea0d858aa0b2cab3f64767